understanding/22/fixed-reference-points.html

```diff
@@ -0,0 +1,131 @@
+         <li>A digital book is published is with no print equivalent and page break locators are inserted which supports direct navigation across platforms and form factors.</li> 
+         <li>Virtual page</li>  
+         <li>Synchronizing reading people with different form factors and platforms, screen sizes, can be on the same page.</li>
+      </ul>
+   
+   </section>
+   
+   <section id="intent">
+        
+      <h2>Intent of Fixed Reference Points</h2>
+
+
+      <p>The purpose of this success criteria is to allow users with disabilities to find references to content based on the page locators found in the default view or printed version of a publication.</p>
+
+      <p>The term “on the same page” is common in English to describe when two people are talking about the same thing and agree on the content of the conversation. This success criteria makes sure that all users can literally be on the same page.</p>
+         
+      <p>Page numbering has long been a fundamental way to identify and communicate the location of written content. They are used constantly in references, footnotes, endnotes, and bibliographies. Particularly, they are critical in academic and learning environments.</p>
+
+      <p>Electronic publishing has provided valuable access to content for people who are blind, have low vision, dyslexia or other cognitive disabilities. In order to consume the information the content may be adapted to use a different  layout, or presented to them using assistive technology.  If there is no clear way to find a specific page from the print version that was referenced by a professor in class, because the electronic version is zoomed-in and that paragraph is on page 145 of the user’s version of the content, the user misses out on valuable and sometimes critical information to understanding the reference.</p>
+
+      <p>This success criteria ensures that users with low vision or those who are blind, will be able to find pages that are referenced based on the default or printed view of the publication.</p>
+		
+		<p class="note">The term “article” in the success criteria does not refer to the <code>&lt;article&gt;</code> tag in PDF or InDesign.</p>
+
+    </section>
+    <section id="benefits">
+      <h2>Benefits of Fixed Reference Points</h2>
+      
+      <ul>
+         <li>People who are blind or who have low vision or those who may be magnifying content because of a cognitive disability will still be able to find their place and find references to content that are based on page numbers of the print or default version of the publication.</li>
+      </ul>
+      
+   </section>
+   
+   <section id="examples">
+      <h2>Examples of Fixed Reference Points</h2>
+
+      <ul>
+         <li>A sortable list of elements may, after focussing a list element, provide adjacent controls for moving the element up or down in the list by simply tapping/clicking on those controls.</li>
```

Comment:
as noted, these examples don't seem to have anything to do with "a web page or set of web pages is an electronic publication with pagebreak locators"